### PR TITLE
Fix storybook CLI flag deprecation warning.

### PR DESCRIPTION
**Pull request by itsaladin**
## Summary

Addresses issue:

- #6569

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -41,4 +41,5 @@ module.exports = {
 
 		return head;
 	},
+	staticDirs: [ '../dist' ],
```
Comment (kuasha420) [inline .storybook/main.js:44]: Nice!

**Review by kuasha420 — APPROVED** [review]
@itsaladin Thanks for contributing to to Site Kit. Changes look good and works. LGTM :+1: 

Moving to Merge Review.

**Comment by kuasha420** [comment]
Looks like the storybook tests were skipped here since this PR is from a fork. I've added a point in the QA:Eng brief to check the tests on Marge commit while QA'ing this. Just making a note here as well.

**Comment by eugene-manuilov** [comment]
@kuasha420, we need to make sure that storybook checks pass in GH. Let's create our own PR instead.

**Pull request closed without merging**